WebDriverAgentLib/Categories/XCElementSnapshot+FBHelpers.m

```diff
@@ -122,6 +75,15 @@ - (BOOL)fb_framelessFuzzyMatchesElement:(XCElementSnapshot *)snapshot
     valuesAreEqual(self.placeholderValue, snapshot.placeholderValue);
 }
 
++ (NSSet<NSNumber *> *)fb_getUniqueTypes:(NSArray<XCElementSnapshot *> *)matchingSnapshots
+{
+  NSMutableSet *matchingTypes = [NSMutableSet set];
+  [matchingSnapshots enumerateObjectsUsingBlock:^(XCElementSnapshot* snapshot, NSUInteger snapshotIdx, BOOL *stopSnapshotEnum) {
```

Comment:
any more details on this? it looks like some hidden objc magic for me and I don't usually use anything that I don't understand.